internal/ingress/controller/checker.go

```diff
@@ -26,6 +26,8 @@ import (
 	"github.com/pkg/errors"
 )
 
+const nginxPID = "/tmp/nginx.pid"
```

Comment:
What about `/run/nginx`? (for some reason I dislike the idea of storing an important files such as pid inside /tmp, it feels counter intuitive to me)